var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -139,10 +142,20 @@ def cmake_args(self):
         else:
             options.append('-DGMX_HWLOC:BOOL=OFF')
 
-        if '+cuda' in self.spec or '+opencl' in self.spec:
-            options.append('-DGMX_GPU:BOOL=ON')
+        if version >= Version('2021'):
```

Comment:
with spack-0.16.0, that line lead to ``Error: TypeError: coerce_versions cannot be called on <class 'function'>``.
Using ``if self.version >= Version('2021'):`` worked fine.

Reply:
Updated to `self.version`